cmd/dex/serve.go

```diff
@@ -98,22 +97,22 @@ func runServe(options serveOptions) error {
 		return fmt.Errorf("invalid config: %v", err)
 	}
 
-	logger.Infof(
-		"Dex Version: %s, Go Version: %s, Go OS/ARCH: %s %s",
-		version,
-		runtime.Version(),
-		runtime.GOOS,
-		runtime.GOARCH,
+	logger.Info(
+		"Version info",
+		"dex_version", version,
+		"go_version", runtime.Version(),
+		"go_os", runtime.GOOS,
+		"go_arch", runtime.GOARCH,
```

Comment:
nit: I wonder if using slog's group feature here would make sense. As far as I understand, that basically means using dot-notated labels (ie. `go.version` instead of `go_version`).

https://pkg.go.dev/log/slog#hdr-Groups

Reply:
I changed it to use groups.
It'll be `go.version` in the text output, and in json `"go": {"version": ...}`